src/components/JobCard/JobCard.tsx

```diff
@@ -1,23 +1,43 @@
+import { Link } from "react-router-dom";
 
+interface JobDetails {
+  id: number;
+  name: string;
+  category: string;
+  job_tags: string[];
+  job_description: string;
+}
+interface JobCardProps {
+  jobDetails: JobDetails;
+}
 
-const JobCard = ({ job }) => {
+const JobCard: React.FC<JobCardProps> = ({ jobDetails }) => {
   return (
-   <div className="max-w-sm rounded overflow-hidden shadow-lg bg-white">
-  <img className="w-full" src="https://via.placeholder.com/400x200" alt="Sunset in the mountains"/>
-  <div className="px-6 py-4">
-    <div className="font-bold text-xl mb-2">The Coldest Sunset</div>
-    <p className="text-gray-700 text-base">
-      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus quia, nulla! Maiores et perferendis eaque, exercitationem praesentium nihil.
-    </p>
-  </div>
-  <div className="px-6 pt-4 pb-2">
-    <span className="inline-block bg-gray-200 rounded-full px-3 py-1 text-sm font-semibold text-gray-700 mr-2 mb-2">#photography</span>
-    <span className="inline-block bg-gray-200 rounded-full px-3 py-1 text-sm font-semibold text-gray-700 mr-2 mb-2">#travel</span>
-    <span className="inline-block bg-gray-200 rounded-full px-3 py-1 text-sm font-semibold text-gray-700 mr-2 mb-2">#winter</span>
-  </div>
-</div>
-     
+    <Link to={`/project/${jobDetails.id}`}>
+      <div className="max-w-sm rounded overflow-hidden shadow-lg bg-white">
+        <img
+          className="w-full"
+          src="https://via.placeholder.com/400x200"
+          alt="Sunset in the mountains"
+        />
+        <div className="px-6 py-4">
+          <div className="font-bold text-xl mb-2">{jobDetails.name}</div>
+          <p className="text-gray-700 text-base">
+            {jobDetails.job_description}
+          </p>
+        </div>
+        <div className="px-6 pt-4 pb-2">
+          {jobDetails.job_tags.map((job_tag: string) => {
+            return (
+              <span className="inline-block bg-gray-200 rounded-full px-3 py-1 text-sm font-semibold text-gray-700 mr-2 mb-2">
+                {job_tag}
```

Comment:
and here

Reply:
The snakecase was used for dummy data, so I needed to use it to access the data for the frontend. hopefully, with the integration of firebase it will be eliminated.